core/domain/topic_domain.py

```diff
@@ -350,6 +395,12 @@ def validate(self):
             ValidationError: One or more attributes of the subtopic are
                 invalid.
         """
+        self.require_valid_thumbnail_filename(self.thumbnail_filename)
```

Comment:
Here and elsewhere -- should check that thumbnail filename + bg color are either "both specified" or "none specified", right?

(Make sure that the frontend doesn't permit "only one is specified" cases, too.)